core/api/workspace.go

```diff
@@ -339,6 +352,17 @@ type ProcessDescription struct {
 	Running  bool   `json:"running"`
 }
 
+type ComponentStatus struct {
+	ComponentID         string            `json:"componentId"`
+	Running             bool              `json:"running"`
+	EnvVars             map[string]string `json:"envVars"`
```

Comment:
We're calling this simply "environment" in many places.

Reply:
My thinking is that "environment" is a little too

Reply:
not sure i follow the distinction you are trying to draw